packages/edit-site/src/components/save-hub/index.js

```diff
@@ -32,29 +55,106 @@ export default function SaveHub() {
 		[]
 	);
 
+	const {
+		editEntityRecord,
+		saveEditedEntityRecord,
+		__experimentalSaveSpecifiedEntityEdits: saveSpecifiedEntityEdits,
+	} = useDispatch( coreStore );
+
 	const disabled = isSaving || ( ! isDirty && ! isPreviewingTheme() );
 
+	// if we have only one unsaved change and it matches current context, we can show a more specific label
+	let dirtyLocal = null;
+
+	if ( countUnsavedChanges === 1 ) {
+		// if we are on global styles
+		if ( params.path?.includes( 'wp_global_styles' ) ) {
+			dirtyLocal = dirtyRecords.find(
+				( record ) => record.name === 'globalStyles'
+			);
+		}
+		// if we are on pages
+		else if ( !! params.postId ) {
```

Comment:
nit: The `!!` is unnecessary since everything within an `if ( )` is evaluated as if it were a boolean.

```suggestion
		else if ( params.postId ) {
```